### PR TITLE
Update pricing page with new freemium offering

### PR DESCRIPTION
**Pull request by Yndira-E**
## Description

This PR introduces the Freemium plan to the pricing page.
This means:
- A fourth card in the hero section of the page
- Brought back the CTA/button for the Starter card
- A new column in the features table
- Updates in some of the Starter and Team features

## Related Issue(s)

https://github.com/FlowFuse/CloudProject/issues/605

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [x] Documentation has been updated

### CODE REVIEW

File: src/pricing/index.njk
Comment (Yndira-E) [inline src/pricing/index.njk:1]: The cards need refactoring; I'll handle that in a separate PR.

**Comment by gstout52** [comment]
I'll merge this tomorrow so the page goes live after the teamtype is made available.

File: src/_data/pricingFeatures.yaml [hunk 1]
```diff
@@ -18,127 +23,122 @@ sections:
...
         selfHostedValues: [ null, 'check' ]    
         tags: [ 'cloud', 'self-hosted' ]
         info: "<p>Connect to an edge device and develop your flows.</p>"
       - id: mqtt-broker-cloud
         label: MQTT Broker
-        cloudValues: [ null, 'check', 'check' ]    
+        cloudValues: [ null, 'check', 'check', 'check' ]    
         tags: [ 'cloud' ]
-        info: "<p>Easily manage and create MQTT clients to transport data for efficient messaging and communication within your applications. The Team Tier includes 5 clients at no extra cost, while the Enterprise Tier includes 20 clients for free as part of your existing plan, with purchase additional clients as needed in the future.</p>"
+        info: "<p>Easily manage and create MQTT clients to transport data for efficient messaging and communication within your applications. At no extra cost, the Starter Tier includes 2 clients, the Team Tier includes 5 clients, and the Enterprise Tier includes 20 clients as part of your existing plan. Additional clients can be purchased as needed in the future.</p>"
```
Comment (ZJvandeWeg) [inline src/_data/pricingFeatures.yaml:65]: Let's remove the at no extra cost part. That should be obvious from the table

File: src/_data/pricingFeatures.yaml [hunk 2]
```diff
@@ -18,127 +23,122 @@ sections:
...
         label: Role-Based Access Control
-        cloudValues: [ 'check', 'check', 'check' ]
+        cloudValues: [ null, 'check', 'check', 'check' ]
         selfHostedValues: [ 'check', 'check' ]
         tags: [ 'cloud', 'self-hosted' ]
         info: "<p>FlowFuse's intuitive team management tooling makes it easy for you to control whom has access to what.</p>"
       - id: security
         label: Endpoint Security
-        cloudValues: [ 'None<br>HTTP Basic Auth', 'None<br>HTTP Basic Auth<br>FlowFuse Authentication', 'None<br>HTTP Basic Auth<br>FlowFuse Authentication' ]
+        cloudValues: [ null, 'None<br>HTTP Basic Auth', 'None<br>HTTP Basic Auth<br>FlowFuse Authentication', 'None<br>HTTP Basic Auth<br>FlowFuse Authentication' ]
```
Comment (ZJvandeWeg) [inline src/_data/pricingFeatures.yaml:111]: The instance do have authentication set up? Cc @knolleary

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/_data/pricingFeatures.yaml (hunk 2 above)
Comment (knolleary) [reply src/_data/pricingFeatures.yaml:111]: @ZJvandeWeg Freemium is Devices only; this row relates to hosted instances

Comment (ZJvandeWeg) [reply src/_data/pricingFeatures.yaml:111]: So the label is incorrect? I know it's edge instance only, but it's not clear?
Now it seems like we're not applying endpoint security.

File: src/_data/pricingFeatures.yaml (hunk 1 above)
Comment (gstout52) [reply src/_data/pricingFeatures.yaml]: Done.

File: src/_data/pricingFeatures.yaml (hunk 2 above)
Comment (gstout52) [reply src/_data/pricingFeatures.yaml:111]: I've just revised the popup to clarify that this row is about hosted instances.

**Review by gstout52 — APPROVED** [review]

**Review by ZJvandeWeg — APPROVED** [review]

**Pull request merged**